app/models/concerns/initiation_procedure_concern.rb

```diff
@@ -0,0 +1,43 @@
+module InitiationProcedureConcern
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def create_initiation_procedure(administrateur)
+      p = Procedure.new(
+        libelle: 'Une première procédure',
+        description: "Une première procédure afin de découvrir les possibilités de #{APPLICATION_NAME}",
+        organisation: 'Dinum',
+        cadre_juridique: "inexistant car c'est un test",
+        duree_conservation_dossiers_dans_ds: 1,
+        for_individual: true,
+        administrateurs: [administrateur]
+      )
+
+      p.draft_revision = p.revisions.build
+
+      p.save!
+
+      p.draft_revision.add_type_de_champ({ type_champ: :text, libelle: 'nouveau champ' })
+      p.defaut_groupe_instructeur.instructeurs << administrateur.instructeur
```

Comment:
petite crainte sur el fait qu'un admin puisse ne pas avoir de compte instructeur ? ou pas ?
```suggestion
      p.defaut_groupe_instructeur.instructeurs << administrateur.instructeur.present? administrateur.instructeur : administrateur.user.build_instructeur
```

Reply:
nop, un admin est toujours un instructeur